src/components/Boxs/BoxBase/index.tsx

```diff
@@ -96,13 +96,15 @@ export interface BoxBaseProps extends MaterialBoxProps {
     minHeight?: number;
     maxHeight?: number;
     overrideColor?: "primary" | "secondary" | "info" | "error" | "warning" | "success";
+    border?: boolean;
 }
 
 const BoxBase: React.FC<BoxBaseProps> = ({
     bgcolor,
     colorProps = "main",
     color,
     overrideColor,
+    border,
```

Comment:
```suggestion
    shouldHasBorder,
```
boolean should start with is or should

Reply:
Ok, it's my mistake!